src/main/kotlin/app/revanced/patches/youtube/misc/integrations/patch/IntegrationsPatch.kt

```diff
@@ -60,6 +73,15 @@ class IntegrationsPatch : BytecodePatch(
                 )
             ).toMutable()
         )
+
+        val serviceIndex = serviceMethod.implementation!!.instructions.lastIndex
+        serviceMethod.addInstructions(
+            serviceIndex, """
+               invoke-static {}, Lcom/google/android/apps/youtube/app/YouTubeTikTokRoot_Application;->getAppContext()Landroid/content/Context;
```

Comment:
That makes no sense here. ReVancedUtils->getContext() returns getAppContext() so I don't see how this would work?

Reply:
uh this instruction returns the context itself so i can  set it in ReVancedUtils

Reply:
Oh ya nvm, I'm dumb lol